# 5.3 参考文献

**[1]** Lindgreen S. AdapterRemoval: easy cleaning of next-generation sequencing reads. BMC Res Notes. 2012, 5:337. 

**[2]** Li H. and Durbin R. Fast and accurate short read alignment with Burrows-Wheeler Transform. Bioinformatics. 2009, 25:1754-60. 

**[3]** Li H, Handsaker B.,Wysoker A., Fennell T.,Ruan J., Homer N., Marth G., et al. The Sequence alignment/map (SAM) format and SAMtools. Bioinformatics, 2009 25:2078-9. 

**[4]** McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, Garimella K, Altshuler D, Gabriel S, Daly M, DePristo MA. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. GENOME RESEARCH. 2010, 20:1297-303. 

**[5]** Wang K, Li M, Hakonarson H. ANNOVAR: Functional annotation of genetic variants from next-generation sequencing data. Nucleic Acids Research. 2010, 38:e164. 

**[6]** Abyzov A, Urban AE, Snyder M, Gerstein M. CNVnator: an approach to discover, genotype, and characterize typical and atypical CNVs from family and population genome sequencing. Genome Res. 2011, 21(6):974-84. 

**[7]** Ken Chen, John W Wallis, Michael D McLellan, David E Larson, Joelle M Kalicki, Craig S Pohl, Sean D McGrath, Michael C Wendl, Qunyuan Zhang, Devin P Locke, Xiaoqi Shi, Robert S Fulton, Timothy J Ley, Richard K Wilson, Li Ding, Elaine R Mardis. BreakDancer: an algorithm for high-resolution mapping of genomic structural variation. Nature Methods. 2009, 677–681.